source/includes/administration/_commitments.md

```diff
@@ -271,6 +281,10 @@ curl "https://cloudmc_endpoint/api/v2/commitments" \
    "pricingMethod": "FIXED_PRICE",
    "type": "RESOURCE",
    "fixedPrice": 500.0,
+   "fixedPriceTaxCode": {
+      "code": "SI086690",
+      "description": "IT Services"
```

Comment:
Is the description going to be mandatory in the API request?

Reply:
It's actually optional, I added an "optional" to the description of the TaxCode object.